tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,625 @@
+
+        delete_additional_resources = true
+
+        autoscaler_config {
+            ignore_daemonsets_utilization = true
+            balance_similar_node_groups   = true
+        }
+
+        auto_upgrade {
+            enable                        = true
+            maintenance_window_day        = "sunday"
+            maintenance_window_start_hour = 2
+        }
+
+        private_network_id = scaleway_vpc_private_network.pn_multi_az.id
+    }
+
+    resource "scaleway_k8s_pool" "pool-multi-az" {
+        for_each = {
+            "fr-par-1" = "PRO2-XXS",
+            "fr-par-2" = "PRO2-XXS",
+            "fr-par-3" = "PRO2-XXS"
+        }
+
+        name                   = "pool-${each.key}"
+        zone                   = each.key
+        tags                   = ["multi-az"]
+        cluster_id             = scaleway_k8s_cluster.kapsule_multi_az.id
+        node_type              = "${each.value}"
+        size                   = 2
+        min_size               = 2
+        max_size               = 3
+        autoscaling            = true
+        autohealing            = true
+        container_runtime      = "containerd"
+        root_volume_size_in_gb = 32
+    }
+    ```
+
+5. To access the Kapsule cluster with `kubectl` you need a `kubeconfig` file, with the address of the cluster and the credentials. Copy/paste this `template_file` and write it into a `kubeconfig` file in the current directory.
+    ```
+    data "template_file" "kubeconfig" {
+        template = <<EOT
+    apiVersion: v1
+    clusters:
+    - name: "$${cluster_name}"
+    cluster:
+        certificate-authority-data: $${ca_cert}
+        server: "$${host}"
+    contexts:
+    - name: admin@kapsule-multi-az
+    context:
+        cluster: "kapsule-multi-az"
+        user: kapsule-multi-az-admin
+    current-context: admin@kapsule-multi-az
+    kind: Config
+    preferences: {}
+    users:
+    - name: kapsule-multi-az-admin
+    user:
+        token: $${token}
+    EOT
+
+        vars = {
+            cluster_name = scaleway_k8s_cluster.kapsule_multi_az.name
+            host         = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].host
+            token        = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].token
+            ca_cert      = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].cluster_ca_certificate
+        }
+    }
+
+    resource "local_file" "kubeconfig" {
+        content  = data.template_file.kubeconfig.rendered
+        filename = "${path.module}/kubeconfig"
+    }
+    ```
+
+6. Initialize your Terraform configuration and apply it to create your resources.
+    ```
+    terraform init
+    terraform apply
+    ```
+
+7. Configure your `KUBECONFIG` environment variable to point to the `kubeconfig` file you just created via Terraform
+    ```
+    export KUBECONFIG=kubeconfig
+    ```
+
+8. List your nodes now to check that your cluster is ready, and that there are two nodes per availability zone.
```

Comment:
```suggestion
5. Initialize your Terraform configuration and apply it to create your resources.
    ```
    terraform init
    terraform apply
    ```

6. To access the Kapsule cluster with `kubectl` you need a `kubeconfig` file, with the address of the cluster and the credentials. You can use the CLI to retrieve and use it.
    ```
    terraform output kapsule
    "fr-par/[CLUSTER-ID]"

    scw k8s kubeconfig install [CLUSTER-ID]
    kubectl config use-context [CONTEXT]
    ```

7. List your nodes now to check that your cluster is ready, and that there are two nodes per availability zone.
    ```

8. List your nodes now to check that your cluster is ready, and that there are two nodes per availability zone.
```